releng/org.eclipse.xtext.target/pom.xml

```diff
@@ -12,4 +12,36 @@
 
 	<packaging>eclipse-target-definition</packaging>
 	<artifactId>org.eclipse.xtext.target</artifactId>
+
+	<build>
```

Comment:
does this verfy the target files on build?

Reply:
or does tycho not verfiy at all?

Reply:
There is no verification. These are just files. Verification happens only when used as target.